### PR TITLE
Moved Desired and Current headers to rs section

### PR DESCRIPTION
**Pull request by rmohta**
"kubectl get rs" has Desired and Current headers. So moved it from Deployment headers explanation section to ReplicaSet.

This should close Issue 20323

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -116,6 +115,13 @@ In this example:
     NAME                          DESIRED   CURRENT   READY   AGE
     nginx-deployment-75675f5897   3         3         3       18s
     ```
+    ReplicaSet output shows the following fields:
+    
+      * `DESIRED` displays the desired number of _replicas_ of the application, which you define when you create the Deployment. This is the _desired state_.
+      * `CURRENT` displays how many replicas are currently running.
+      * `READY` displays how many replicas of the application are available to your users.
+      * `AGE` displays the amount of time that the application has been running.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/deployment.md:123]: I would add `NAME` as well. Could you revise to include that @rmohta? If you know how to squash your commits, please do that too.

Comment (rmohta) [reply content/en/docs/concepts/workloads/controllers/deployment.md:124]: Done. Commits have also been squashed.

**Comment by rmohta** [comment]
Link GH-20323

**Pull request merged**